task/02-numbers-tasks.js

```diff
@@ -22,10 +21,9 @@
  *   5, 5  => 25
  */
 function getRectangleArea(width, height) {
-    throw new Error('Not implemented');
+  return width * height;
```

Comment:
JFYI: calculate value of expressions, save it to variable and return variable. Intel/Google styleguide.